source/_integrations/iotty.markdown

```diff
@@ -30,14 +31,14 @@ This integration currently supports the following iotty devices:
 
 - iotty Smart Switch \
   US version:
-  - [1-Switch Controller](https://iottysmarthome.com/products/1-switch-controller?variant=43630747058389)
-  - [2-Switch Controller](https://iottysmarthome.com/products/2-switch-controller?variant=43630751219925)
-  - [3-Switch Controller](https://iottysmarthome.com/products/3-switch-controller?variant=43630760493269)
-  - [4-Switch Controller](https://iottysmarthome.com/products/4-switch-controller?variant=43630774386901)
+  - [1-Switch Controller](https://link.iotty.com/DU-1-gang-smart-switch)
```

Comment:
What is the difference?

Reply:
If you look at the first time this documentation was submitted, it is a bit old. We have changed the links in our website since then, so now we are using the new ones.

Reply:
Can I ask what is the issue with redirects? I couldn't find any mention about that in the documentation contribution guidelines

Reply:
There is no need to redirect users unneededly.